src/components/lib/cos_defcompinfo_api.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2016, Phani Gadepalli and Gabriel Parmer, GWU, gparmer@gwu.edu.
+ *
+ * This uses a two clause BSD License.
+ */
+
+#include <cos_defcompinfo_api.h>
+
+static struct cos_defcompinfo curr_defci;
+
+struct cos_defcompinfo *
+cos_defcompinfo_curr_get(void)
+{ return &curr_defci; }
```

Comment:
Is there any way we can add an assertion to check that curr_defci has been initialized?  If so, add a comment above the assertion; something like `/* You must call cos_defcompinfo_init first */`.